ci/setup-build-test-gdk.ps1

```diff
@@ -4,32 +4,34 @@ param(
     [string] $msbuild_exe = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\MSBuild.exe",
     [string] $build_home = (Get-Item "$($PSScriptRoot)").parent.parent.FullName, ## The root of the entire build. Should ultimately resolve to "C:\b\<number>\".
     [string] $unreal_engine_symlink_dir = "$build_home\UnrealEngine",
-    [string] $gyms_version_path = "$gdk_home\UnrealGDKTestGymsVersion.txt"
+    [string] $gyms_version_path = "$gdk_home\UnrealGDKTestGymVersion.txt",
+    [string] $engine_net_version_path = "$gdk_home\UnrealGDKEngineNetTestVersion.txt"
 )
+. "$PSScriptRoot\common.ps1"
 
 class TestProjectTarget {
     [ValidateNotNullOrEmpty()][string]$test_repo_url
     [ValidateNotNullOrEmpty()][string]$test_repo_branch
     [ValidateNotNullOrEmpty()][string]$test_repo_relative_uproject_path
     [ValidateNotNullOrEmpty()][string]$test_project_name
     
-    TestProjectTarget([string]$test_repo_url, [string]$gdk_branch, [string]$test_repo_relative_uproject_path, [string]$test_project_name, [string]$test_gyms_version_path, [string]$test_env_override) {
+    TestProjectTarget([string]$test_repo_url, [string]$gdk_branch, [string]$test_repo_relative_uproject_path, [string]$test_project_name, [string]$test_version_path, [string]$test_env_override) {
         $this.test_repo_url = $test_repo_url
         $this.test_repo_relative_uproject_path = $test_repo_relative_uproject_path
         $this.test_project_name = $test_project_name
         
         # Resolve the branch to run against. The order of priority is:
         # envvar > same-name branch as the branch we are currently on > UnrealGDKTestGymVersion.txt > "master".
         $testing_repo_heads = git ls-remote --heads $test_repo_url $gdk_branch
-        $test_gym_version = if (Test-Path -Path $test_gyms_version_path) { [System.IO.File]::ReadAllText($test_gyms_version_path) } else { [string]::Empty }
+        $test_version = if ([System.IO.File]::Exists($test_version_path)) {[System.IO.File]::ReadAllText($test_version_path)} else {[string]::Empty}
```

Comment:
Nit: test_version seems like a bit of a weird name, but not sure what to replace it with

Reply:
text_file_version_override?

Reply:
that does sound better, I think